### DISCUSSION TITLE
Helping fight COVID-19 using GitHub

### DISCUSSION THREAD

**Discussion by AnandChowdhary** [github/india#51] [Open Source]
👋 @richakumar and I are the panelists from the session [Helping fight COVID-19 using GitHub](https://githubsatellite.com/sessions/helping-fight-covid-19-using-github/) in the open source track. Let's use this Discussion for any questions!

Here are some links:

- Karuna 2020 website: https://karuna2020.org
- Karuna 2020 on GitHub: https://github.com/Karuna2020
- [How Clojure's concurrency model helped us feed 12000 people](https://krimlabs.com/blog/clojure-core-async-to-sustain-12k) by @shivekkhurana
- [Karuna 2020: Using GitHub Actions in the COVID-19 response efforts](https://github.blog/2020-08-13-github-actions-karuna/) by @MishManners
- COVID-19 topic on GitHub: https://github.com/topics/covid-19
- My personal website: https://anandchowdhary.com

**Comment by tsadarsh** [comment]
Awesome talk. This is so inspiring :smile:

**Reply by richakumar** [reply]
Remember what Anand said, each one of us can contribute, we just need to start 😄.

**Comment by manavmodi22** [comment]
Great Insights!
1. Building on top of Open Source
2. Github Actions-based automation.
3. Zero Server Cost using Github Pages
4. Open Sourcing the project itself to help build other non-profits on top of it!
5. Promoting Transparency while preserving the privacy of people involved through open source projects!

**Reply by AnandChowdhary** [reply]
This is a great overview, thanks! I'm going to steal it for my talk summary! 😉

**Reply by tsadarsh** [reply]
Another perk of open-source :laughing:

**Comment by richakumar** [comment]
You can also check out some of the other open source projects dedicated to helping the COVID-19 relief effort at: https://github.blog/2020-03-23-open-collaboration-on-covid-19/

**Reply by manavmodi22** [reply]
Great help! Such blogs are really a great source of inspiration to start projects! Building on top of open source projects really improves the projects developed.

**Comment by aminoxix** [comment]
> 👋 @richakumar and I are the panelists from the session [Helping fight COVID-19 using GitHub](https://githubsatellite.com/sessions/helping-fight-covid-19-using-github/) in the open source track. Let's use this Discussion for any questions!
> 
> Here are some links:
> 
> * Karuna 2020 website: https://karuna2020.org
> * Karuna 2020 on GitHub: https://github.com/Karuna2020
> * [How Clojure's concurrency model helped us feed 12000 people](https://krimlabs.com/blog/clojure-core-async-to-sustain-12k) by @shivekkhurana
> * [Karuna 2020: Using GitHub Actions in the COVID-19 response efforts](https://github.blog/2020-08-13-github-actions-karuna/) by @MishManners
> * COVID-19 topic on GitHub: https://github.com/topics/covid-19
> * My personal website: https://anandchowdhary.com

Great!
